deploy/crds/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -18,6 +18,18 @@ metadata:
   namespace: mlnx-network-operator
 spec:
   ofedDriver:
-    image: "ofed-driver"
-    repository: "adriancdockerhub"
-    version: "5.0-2.1.8.1"
+    image: ofed-driver
+    repository: adriancdockerhub
+    version: 5.0-2.1.8.1
+  devicePlugin:
+    image: k8s-rdma-shared-dev-plugin
+    repository: rdma
+    version: latest
```

Comment:
what does latest mean? master?

Reply:
seeing there are no releases, its master but the last push to dockerhub was 25 days ago while the last commit was 16 days ago, so i guess something is not in sync.

we need to have a release for it and a proper tag.

This however is not related to this commit.
i suggest as above to replace with some generic value that hints the user to replace it for now